content/en/infrastructure/containers/orchestrator_explorer.md

```diff
@@ -140,17 +140,220 @@ All of these columns support sorting, which helps you to pinpoint individual wor
 
 {{< img src="infrastructure/livecontainers/orch_ex_resource_utilization_sorted_column.png" alt="Container Resource Utilization Sorted Columns" style="width:50%;">}}
 
+## Query filter details
+
+You can narrow down the displayed resources by supplying a query within the "Filter by" search bar on the top left of the page.
+
+### Syntax
+
+A query filter is composed of terms and operators. Example:
+
+{{< img src="infrastructure/livecontainers/orch_syntax.png" alt="Orchestrator Explorer query filter syntax." style="width:80%;">}}
+
+#### Terms
+
+There are multiple types of terms available:
+
+| Type | Examples |
+|---|---|
+| **Tags**: Attached to resources by [the agent collecting them](https://docs.datadoghq.com/getting_started/tagging/assigning_tags/?tab=containerizedenvironments). There are also additional tags that Datadog generates for Kubernetes resources. | `datacenter:staging`<br>`tag#datacenter:staging`<br>*(the `tag#` is optional)* |
```

Comment:
```suggestion
| **Tags**: Attached to resources by [the agent collecting them](https://docs.datadoghq.com/getting_started/tagging/assigning_tags/?tab=containerizedenvironments). There are also additional tags that Datadog generates for Kubernetes resources. | `datacenter:staging`<br>`tag#datacenter:staging`<br>*(the `tag#` is optional)* |
```

Ah I missed all the links in this table/section, need to be annotated as mentioned before : https://datadoghq.atlassian.net/wiki/spaces/docs4docs/pages/1182958054/Markdown#Links